cob_phidget_power_state/scripts/power_state_phidget.py

```diff
@@ -1,23 +1,24 @@
 #!/usr/bin/env python
 
-import rospy
 import numpy as np
-from cob_phidgets.msg import AnalogSensor
+import rospy
 from cob_msgs.msg import PowerState
+from cob_phidgets.msg import AnalogSensor
+
 
 class PowerStatePhidget():
     PHIDGET_MAX_VALUE = 999
     PHIDGET_MIN_VALUE = 0
     PERIOD_RECORD_SIZE = 6
-    VOLTAGE_COLLECTION_TIME = 6.0 #sec
+    VOLTAGE_COLLECTION_TIME = 6.0  # sec
 
     def __init__(self):
         self.voltage = None
         self.current = None
         self.last_update = rospy.Time(0)
         self.charging = False
 
-        self.voltage_max = rospy.get_param("~voltage_max", 59.5)
+        self.voltage_divider_factor = rospy.get_param("~voltage_divider_factor", 59.5)
```

Comment:
if this is no more a maximum voltage is there a better default value? If not, I would make this a non-optional parameter and thow an error if missing.